.github/workflows/test.yml

```diff
@@ -168,50 +173,69 @@ jobs:
         if: always()
         shell: bash
         env:
-          GH_TOKEN: ${{ github.token }}
           COV_STATUS: ${{ env.COV_STATUS }}
+          GH_TOKEN: ${{ github.token }}
+          ORG_REPO: ${{ github.repository }}
         run: |
           PR_NUM=$(jq -r '.number' "$GITHUB_EVENT_PATH")
-          comments=$(gh api -X GET /repos/${{ github.repository }}/issues/${PR_NUM}/comments)
-          comment_id=$(echo "$comments" | jq '.[] | select(.user.login == "github-actions[bot]" and (.body | tostring | contains("<!-- COVERAGE_COMMENT_'${PR_NUM}' -->"))) | .id')
-          coverage_label=$(gh pr view "$PR_NUM" --json labels --jq ' .labels[].name | select(. == "coverage-diff")')
-          if [[ -n "$comment_id" ]]; then
-            gh api -X DELETE /repos/${{ github.repository }}/issues/comments/"$comment_id"
+          COMMENTS=$(gh api -X GET /repos/"${ORG_REPO}"/issues/"${PR_NUM}"/comments)
+          COMMENT_ID=$(echo "$COMMENTS" | jq '.[] | select(.user.login == "github-actions[bot]" and (.body | tostring | contains("<!-- COVERAGE_COMMENT_'${PR_NUM}' -->"))) | .id')
+          COVERAGE_LABEL=$(gh pr view "$PR_NUM" --json labels --jq ' .labels[].name | select(. == "coverage-diff")')
+          if [[ -n "$COMMENT_ID" ]]; then
+            gh api -X DELETE /repos/"${ORG_REPO}"/issues/comments/"${COMMENT_ID}"
           fi
-          if [[ $COV_STATUS != '0' ]]; then
-            if [[ $coverage_label == "coverage-diff" ]]; then
+          if [[ "$COV_STATUS" != '0' ]]; then
+            if [[ "$COVERAGE_LABEL" == "coverage-diff" ]]; then
               gh pr edit "$PR_NUM" --remove-label "coverage-diff"
             fi
           fi
 
       - name: Add diff coverage report comment to PR
         if: always()
         env:
-          GH_TOKEN: ${{ github.token }}
+          BRANCH_NAME: ${{ github.base_ref }}
           COV_STATUS: ${{ env.COV_STATUS }}
+          GH_TOKEN: ${{ github.token }}
```

Comment:
it doesn't look like you're actually using `GH_TOKEN` anywhere. maybe it's used under the hood by some of the commands

Reply:
It's required for `gh` cli